### ISSUE TITLE
Widget request - App launch bar

### ISSUE THREAD

**Issue by magusonline** [tjmarkham/win10widgets#13]
Does that make sense? I'm sure there are a bunch out there, but would like one themed to this. If you can explain how you went about the theming, I could probably just do it myself too. 

Or add shortcuts, etc to it.

**Comment by tjmarkham** [comment]
Can you link to an example or a screenshot of something similar? I'm not exactly sure I know what you mean.

**Comment by magusonline** [comment]
Like a bar like that that you could extend the number of rows

http://imgur.com/a/GJIL6

The main reasoning behind my request is because I use 3 monitors, and they don't allow me to pin shortcuts onto separate task bars

**Comment by tjmarkham** [comment]
I'm not sure what theming would really be necessary... you can just put a bunch of icons on a semi-transparent grey rectangle and that's pretty much it (unless I'm missing something). If that is the case, the RBGA background color used in Win10 Widgets is "24,24,24,217"

**Comment by tjmarkham** [comment]
Since this is essentially duplicating the functionality of the Windows taskbar, I'm not going to be pursuing this.

**Comment by tjmarkham** [comment]
Thought about this a bit. A customizable launcher "grid" to display icons for launching programs could potentially be useful and would be different enough from the taskbar to warrant further investigation.

Essentially, the user would set the number of rows and columns and then be able to click each grid location to set the program for that location. Not sure if this is technically feasible or not, but I'm going to re-open this on the basis that the idea has merit.

**Comment by magusonline** [comment]
Yes! This is exactly what I was meaning! Where you could create a grid that could hot launch shortcuts! Hopefully it's doable.

**Comment by IrishJayhawker** [comment]
I would like to express an interest in this feature also. Something like this would be awesome. 

http://windows7themes.net/en-us/windows-7-app-launcher-gadget/ 

I have folders for things like games, office, internet shortcuts, etc.. but hate having to open a folder to just click on a shortcut but I also do not want my desktop to have 100+ icons on it. 

Ideally it would be small and I would use my mouse wheel to access icons that do not fit on the windowed section. kind of like this: 

https://gyazo.com/a5a31a4357d33cf31755d09129537417

Thanks for listening(reading) and keep up the awesome work!

-IrishJayhawker

**Comment by Crsmsg** [comment]
I'm interest to this feature also.
It will be more quick for one click on the icon from widget and run apps,I need this for organize my exe that i use to control my home appliance on my secondary monitor
